app/src/main/scala/org/alephium/explorer/persistence/queries/ContractQueries.scala

```diff
@@ -66,6 +81,36 @@ object ContractQueries {
     }
   }
 
+  private def updateContractDestructionEventEntities(
+      destroyInfos: Iterable[ContractEntity.DestroyInfo]): DBActionW[Int] = {
+    QuerySplitter.splitUpdates(rows = destroyInfos, columnsPerRow = 5) { (destroyInfos, _) =>
```

Comment:
Does it skip DB insertion when the `Iterable` is empty?

Reply:
yes if it's empty DBAction is 
```
 DBIOAction.successful(0)
```
from [here](https://github.com/alephium/explorer-backend/blob/741f9996ee7d58f0bd859680603a0c1fce70b8e5/app/src/main/scala/org/alephium/explorer/persistence/queries/QuerySplitter.scala#L41)
It's in [this part of the code](https://github.com/alephium/explorer-backend/blob/741f9996ee7d58f0bd859680603a0c1fce70b8e5/app/src/main/scala/org/alephium/explorer/persistence/queries/QuerySplitter.scala#L122-L140) that if [the rows are empty ](https://github.com/alephium/explorer-backend/blob/741f9996ee7d58f0bd859680603a0c1fce70b8e5/app/src/main/scala/org/alephium/explorer/persistence/queries/QuerySplitter.scala#L123-L124) we return the above

Reply:
Great.